code/modules/mob/dead/observer/observer.dm

```diff
@@ -206,12 +206,25 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 // This is the ghost's follow verb with an argument
 /mob/dead/observer/proc/ManualFollow(atom/movable/target)
-	if(target && target != src)
-		if(orbiting && orbiting == target)
-			return
+	if (!istype(target))
+		return
+
+	var/icon/I = icon(target.icon,target.icon_state,target.dir)
+
+	var/orbitsize = (I.Width()+I.Height())/2
+	orbitsize -= (orbitsize/32)*8

```

Comment:
well if you're gonna leave it at `/32` make it `*0.03125` (leave a comment indicating it's functionally `/32`) to help speed things up